dashboard/app/controllers/activities_controller.rb

```diff
@@ -122,10 +122,10 @@ def milestone
         track_progress_in_session
       end
 
-      # If a student in the pilot is submitting work on an AI-enabled level, and their teachers haven't opted-out, trigger the AI evaluation job.
-      is_ai_experiment_enabled = current_user && Experiment.enabled?(user: current_user, script: @script_level&.script, experiment_name: 'ai-rubrics') && Policies::Ai.ai_rubrics_enabled_for_script_level?(current_user, @script_level)
+      # If a student is submitting work on an AI-enabled level, and their teachers haven't opted-out, trigger the AI evaluation job.
+      is_ai_enabled = current_user && Policies::Ai.ai_rubrics_enabled_for_script_level?(current_user, @script_level)
```

Comment:
it looks like we're leaving behind some logic to check that the student belongs to a section that is assigned to the current script level. I can't think of any reason we would want to do this. @wilkie , do you have any context for whether this might still make sense now that we are launching the experiment? I found this PR: https://github.com/code-dot-org/code-dot-org/pull/58061 CC @tess323

Reply:
OK, I checked this, and the ai_rubrics_enabled_for_script_level is the right way to go. I added a verfied_teacher check to ai_rubrics_enabled, and ai_rubrics_enabled_for_script_level runs that for each section that includes the student and script_level in the argument. If there's a better way to check whether a student's teacher has AI enabled when the student submits an assignment, let me know!

Reply:
I think there is some room for interpretation here, but these are the lines I'm concerned about: https://github.com/code-dot-org/code-dot-org/blob/a0b4979ddd3287e5bdf489e9327bf46ce66831b0/dashboard/lib/policies/ai.rb#L19-L20

this logic seems a bit restrictive. one common case could be that if the teacher assigns the CSD course (or focus-on-coding, or focus-on-creativity) but does not assign a particular unit, their students will not automatically have AI run when they submit their projects.

one naive alternative is to run AI if the student has any teacher who has not disabled AI, but that seems too permissive.

after considering the alternatives I think this is fine to leave as-is. heads up @tess323 that this could very well be a common cause of teachers having to manually run ai against their student projects, and that we might want to add more sophisticated logic for when to do so (needs product/engineering design).

Reply:
see https://codedotorg.slack.com/archives/C050HE4QDHR/p1726067444733629 for follow up